src/Sylius/Bundle/AdminBundle/Twig/Component/Shared/Navbar/UserDropdownComponent.php

```diff
@@ -65,16 +65,19 @@ public function getMenuItems(): array
                 'title' => 'sylius.ui.documentation',
                 'url' => 'https://docs.sylius.com',
                 'class' => 'small text-muted',
+                'attributes' => { 'target' => '_blank', 'rel' => 'noopener' },
```

Comment:
```suggestion
                'attributes' => ['target' => '_blank', 'rel' => 'noopener'],
```
and the same below

Reply:
Oups my bad I mix twig and PHP in my suggestion...